src/SingleReport/SingleGamepad.h

```diff
@@ -47,9 +47,15 @@ class SingleGamepad_ : public PluggableUSBModule, public GamepadAPI
     
     virtual void SendReport(void* data, int length) override;
 };
+
+/* These are defined in individual files (SingleGamepadX.cpp) so that every
+ * instance is only created by the compiler if it is references.
+ *
+ * See #266 for details.
+ */
 extern SingleGamepad_ Gamepad1;
 extern SingleGamepad_ Gamepad2;
 extern SingleGamepad_ Gamepad3;
 extern SingleGamepad_ Gamepad4;
-
-
+extern SingleGamepad_ Gamepad5;
+extern SingleGamepad_ Gamepad6;
```

Comment:
If I remember correct, only 3 will ever work, as the 32u4 only has 6 usb endpoints, of which 3 are used by the serial?

Reply:
Correct, but they turn up handy if the CDC serial gets disabled. Since with this modification their presence won't cause any overhead, I thought I'd throw them in.